src/tsweek/tsweek.entity.ts

```diff
@@ -40,10 +40,10 @@ export class TsWeek extends BaseEntity {
   monthNo: number
 
   @ApiProperty()
-  @Column({ name: 'begin' })
+  @Column({ type: 'timestamptz', name: 'begin' })
```

Comment:
Nice - I had not seen this type before - I presume it means timestamp with timezone. I think though, we should be setting the timezone of the database and the machine hosting it to UTC (i.e. 0 hours offset and no daylight savings time)

Reply:
I spent some time looking into the time stamps with timezones. I was just warping my head around it. Plus putting date is not setting the same dates as it was before. I came across an explanation and as far as I understand this does not use local timezone